api/src/test/java/org/pmiops/workbench/db/dao/InstitutionEmailDomainDaoTest.java

```diff
@@ -0,0 +1,104 @@
+package org.pmiops.workbench.db.dao;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.pmiops.workbench.db.model.DbInstitution;
+import org.pmiops.workbench.db.model.DbInstitutionEmailDomain;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.annotation.DirtiesContext.ClassMode;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+@DirtiesContext(classMode = ClassMode.BEFORE_EACH_TEST_METHOD)
+public class InstitutionEmailDomainDaoTest {
+
+  @Autowired InstitutionDao institutionDao;
+  @Autowired InstitutionEmailDomainDao institutionEmailDomainDao;
+
+  private DbInstitution testInst;
+
+  @Before
+  public void setUp() {
+    testInst =
+        institutionDao.save(
+            new DbInstitution().setShortName("Broad").setDisplayName("The Broad Institute"));
+  }
+
+  @Test
+  public void test_getByInstitution_empty() {
+    assertThat(institutionEmailDomainDao.getByInstitution(testInst)).isEmpty();
+    assertThat(institutionEmailDomainDao.count()).isEqualTo(0L);
+  }
+
+  @Test
+  public void test_getByInstitution_multiple() {
+    final DbInstitutionEmailDomain one =
```

Comment:
nit: the spelled out numbers as names confuse me. They make me use the wrong side of my brain.